Can't build esp-matter examples (VSC-981)

### OS

Linux

### Operating System version

Ubuntu

### Visual Studio Code version

Lastest

### ESP-IDF version

4.4.2

### Python version

3.8.7

### Doctor command output

[Not checked]

### Extension

_No response_

### Description

When installing ESP-Matter, building available examples from the repo results on fail.

### Debug Message

```plain
CMake Error at /workspaces/esp-matter/connectedhomeip/connectedhomeip/config/esp32/components/chip/CMakeLists.txt:230 (message):
  The 'gn' command was not found.  Make sure you have GN installed.Or have
  followed necessary build preparations stated in BUILDING.md.
```


### Other Steps to Reproduce

_No response_

### I have checked existing issues, online documentation and the Troubleshooting Guide

- [X] I confirm I have checked existing issues, online documentation and Troubleshooting guide.

--- a/src/utils.ts
+++ b/src/utils.ts
@@ -913,6 +913,7 @@ export function appendIdfAndToolsToPath(curWorkspace: vscode.Uri) {
       "connectedhomeip",
       ".environment",
       "cipd",
+      "packages",
       "pigweed"
     );
     modifiedEnv.ESP_MATTER_DEVICE_PATH = path.join(

